src/js/core.js

```diff
@@ -1145,6 +1145,15 @@
             }
         },
 
+        getContent: function (index) {
+            index = index || 0;
+
+            if (this.elements[index]) {
+                return this.elements[index].innerHTML.trim();
```

Comment:
Yeah, `serialize()` returns the value of all of the elements, so you're right this is just a helper to avoid getting the html for all the elements.  I could change serialize to to call `getContent()` once for each element, but it didn't seem like much of a gain...